addons/context_actions/functions/fnc_healPlayers.sqf

```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: 3Mydlo3
+ * Heals players in given selection.
+ *
+ * Arguments:
+ * N: Selected Objects <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player1, player2, player3] call zen_context_actions_fnc_healPlayers
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _players = _this select {isPlayer _x};
+
+private _moduleGroup = createGroup sideLogic;
+{
+	private _module = _moduleGroup createUnit [
+		QEGVAR(modules,moduleHeal),
```

Comment:
This will throw a bunch of error messages for all non-`CAManBase` units.

A better way to do this would be to move the healing logic to some common function, then calling that directly from here.

Reply:
I thought about healing logic transfer, but didn't want to do it unless told to. I'll make common function variant then.